sepa_sale_order_autoconfirm/models/sale_order.py

```diff
@@ -0,0 +1,17 @@
+from odoo import models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def action_quotation_sent(self):
+        super().action_quotation_sent()
+        website_sale = self.team_id == self.env.ref(
+            "sales_team.salesteam_website_sales"
+        )
```

Comment:
is this to detect whether this is done through the e-commerce? this seems a little dirty to me. isn’t there a cleaner way? like checking whether `self.website_id` is truthy?

Reply:
fixed